### DISCUSSION TITLE
Beginner questions

### DISCUSSION THREAD

**Discussion by oxraap** [Audiveris/audiveris#752] [General]
My initial thoughts on Audiveris:
-Wow, this is a competent software.
-Wow, this software has a seriously steep learning curve.

First questions:
How do I re-assign a chord to a different voice? (i.e. make all notes and rests purple in the attached image)
Can I "flatted" the book to become single voice?
Related question: How do I tell the software to consider the score to be single voice, before commencing scanning/analysis.

![Screenshot - 2024-09-03 , 22_28_01](https://github.com/user-attachments/assets/8c51004a-2a2d-4016-ba45-20c9a5dc6951)

**Comment by hbitteur** [comment]
> How do I tell the software to consider the score to be single voice, before commencing scanning/analysis.

There is no way to do this in the current version.
But this is a very good idea. I'm putting it on the todo list.

**Comment by hbitteur** [comment]
> How do I re-assign a chord to a different voice? (i.e. make all notes and rests purple in the attached image)

The voice menu is your friend, see the handbook [dedicated section](https://audiveris.github.io/audiveris/_pages/ui_tools/chords/).

You can force the voice number for a chord, but only for the first time slot of a measure. This is generally not the way to go.

The best way to guide voice assignments is to declare two selected chords as:
- "next in voice" / "separate voices"
- "same time slot" / "separate time slots"

These user commands are just a guidance. To be effective, the various chords must be correctly recognized.
For instance, in your example, the 2nd measure looks strange. The first note (D5) appears in pink, not the same color as the following note (A4), although they should be part of the same beam group, and thus the same voice. 
Also the following augmentation dot should relate to the D5 note, and thus appear in the same color, which is not the case.

Regarding the last measure, it contains several rests (whole, breve, long) which are each supposed to last the whole measure duration.
Therefore, they cannot belong to the same voice, unless you explicitly set the flag "multi-whole head chords" in the book parameters.

**Comment by hbitteur** [comment]
> -Wow, this software has a seriously steep learning curve.

I agree. Some practice is needed.
I'd advise you to carefully read the detailed [Trinitas example](https://audiveris.github.io/audiveris/_pages/ui_examples/trinitas/README/) in Audiveris handbook.

**Reply by eefweenink** [reply]
dead link. 
this seems the correct link: https://audiveris.github.io/audiveris/_pages/guides/ui/ui_examples/trinitas/README/

**Comment by EddieM0710** [comment]
I have just installed Audiveris. I am very impressed. I have done enough software development to know that there has been a lot of time invested here. It does have a steep learning curve which good documentation will improve. In my experience the documentation is as hard to do right as the development. So, thanks for your effort so far.
Anyway, my question… I’ve installed on windows and it does translate the music but none of the text; title or lyrics. I assume that means that I don’t have some ocr files installed correctly but I don’t know where to look or what to look for. Can you give me any pointers as to what to try?
Thanks,
Eddie

**Reply by EddieM0710** [reply]
Well, never mind about that. I had not installed the ocr data. I thought I had.
I’m still very impressed. I just have to climb that learning curve.
Eddie
